styling/Phoenix.html

```diff
@@ -35,7 +35,23 @@
+--></div></div></div></div></header>
+
+
+<div class="container body-container"><div><div id="subheader-columns"><div class="row">
+<div class="col-xs-4 subheader"><h4>How is Phoenix different?</h4>
+<p>Phoenix is a framework for building HTML5 apps, API backends and distributed systems. Written in Elixir, you get a beautiful syntax, a productive tooling and fast runtime.
+<a href="http://www.phoenixframework.org/v0.10.0/docs">Get started with Phoenix</a></p></div>
+<div class="col-xs-4 subheader"><h4>Building the new web</h4>
+<p>Channels provide real-time streaming directly within Phoenix, allowing us to build rich, interactive applications, be it in the browser, a native mobile app or in embedded devices.
+<a href="http://www.phoenixframework.org/v0.10.0/docs">Learn about channels</a></p></p></div>
+<div class="col-xs-4 subheader"><h4>Battle-proven technology</h4>
+<p>Phoenix leverages the Erlang VM ability to handle millions of connections concurrently while also enjoying the VM guarantees for building fault-tolerant systems.
```

Comment:
`<strong>handle millions of connections concurrently<strong>` `<strong>guarantees for building fault-tolerant systems</strong>` 

These are all suggestions for what I think is most important, but feel free to disregard :)